src/main/scala/io/iohk/ethereum/timing/Beacon.scala

```diff
@@ -0,0 +1,63 @@
+package io.iohk.ethereum.timing
+
+import akka.actor.{Actor, ActorLogging, ActorRef, Scheduler}
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+
+
+class Beacon(
+  miner: ActorRef,
+  slotDuration: FiniteDuration,
+  systemStartTime: Long = System.currentTimeMillis,
```

Comment:
Minor detail up to discussion: Maybe we can rename this to `systemStartTimeMillis` or use `FiniteDuration` as well here?

Reply:
As we discussed with Nico and Mirko, it looks rather odd to implement a point in time as a `FiniteDuration`. However, the suggested changes will be implemented for the sake of consistency. Also, it was made clear that points in time are classic Unix epoch times.

Reply:
Done.